### PR TITLE
Load core on console as well + minor bug fixes and improvements.

### PR DESCRIPTION
**Pull request by cmfcmf**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | #2883
| Refs tickets      | ---
| License           | MIT
| Changelog updated | no

### CODE REVIEW

**Comment by craigh** [comment]
@cmfcmf please correct the StyleCI issues with this PR

File: src/lib/bootstrap.php [hunk 1]
```diff
@@ -11,74 +11,23 @@
...
 if (is_readable($file = __DIR__.'/../app/config/custom_parameters.yml')) {
     $kernelConfig = array_merge($kernelConfig, Yaml::parse(file_get_contents($file)));
 }
 $kernelConfig = $kernelConfig['parameters'];
-if ($kernelConfig['env'] !== 'prod') {
-    // hide deprecation errors
-    // @todo remove exclusions for Core-2.0
-    Debug::enable(E_ALL & ~E_USER_DEPRECATED);
+if ($kernelConfig['debug'] == true) {
+    Debug::enable();
```
Comment (craigh) [inline src/lib/bootstrap.php]: why remove the flags?

**Comment by craigh** [comment]
@cmfcmf please complete this 😉 

also needs a changelog probably.

also - I think the `zikula.php` file might be renamed to `zikula_autoload.php` or similar. To clarify that it is the autoloader? thoughts?

File: src/lib/bootstrap.php (hunk 1 above)
Comment (cmfcmf) [inline src/lib/bootstrap.php]: The flags were there as workaround for a Symfony bug which isn't present any longer. The change of the condition is to comply more with a standard Symfony environment.

**Comment by craigh** [comment]
some issues with travis for you to fix ⛵

File: src/app/autoload.php [hunk 2]
```diff
@@ -26,26 +26,30 @@
...
-class_alias('\Zikula\Bundle\HookBundle\Hook\DisplayHook', '\Zikula\Core\Hook\DisplayHook');
-class_alias('\Zikula\Bundle\HookBundle\Hook\DisplayHookResponse', '\Zikula\Core\Hook\DisplayHookResponse');
-class_alias('\Zikula\Bundle\HookBundle\Hook\FilterHook', '\Zikula\Core\Hook\FilterHook');
-class_alias('\Zikula\Bundle\HookBundle\Hook\Hook', '\Zikula\Component\HookDispatcher\Hook');
-class_alias('\Zikula\Bundle\HookBundle\Hook\ProcessHook', '\Zikula\Core\Hook\ProcessHook');
-class_alias('\Zikula\Bundle\HookBundle\Hook\ValidationHook', '\Zikula\Core\Hook\ValidationHook');
-class_alias('\Zikula\Bundle\HookBundle\Hook\ValidationProviders', '\Zikula\Core\Hook\ValidationProviders');
-class_alias('\Zikula\Bundle\HookBundle\Hook\ValidationResponse', '\Zikula\Core\Hook\ValidationResponse');
+static $aliasesCreated = false;
+if (!$aliasesCreated) {
```
Comment (craigh) [inline src/app/autoload.php]: why/how is it attempting to create them twice now?

**Comment by cmfcmf** [comment]
> some issues with travis for you to fix

I've now updated and fixed it once again.

**Comment by craigh** [comment]
should the aliases be in one of the files indicated as _legacy_ (since they are going to be removed?)

**Comment by cmfcmf** [comment]
> should the aliases be in one of the files indicated as legacy (since they are going to be removed?)

I think they _must_ go into `autoload.php` or a file required in there. We can always move them into an extra file if you want, but I don't feel the need.

**Comment by craigh** [comment]
> but I don't feel the need.

merge it then. 👍

**Pull request merged**